lib/stat_tracker.rb

```diff
@@ -1,5 +1,3 @@
-require './lib/class_helper'
```

Comment:
This should be required in stat_tracker.rb, not the runner. StatTracker directly needs to reference these things, not the runner.